libyara.NET/Scanner.h

```diff
@@ -92,6 +92,8 @@ namespace libyaraNET {
                     resultsHandle.GetPointer(),
                     TimeoutSeconds));
 
+            CloseHandle(fd);
```

Comment:
This code looks like it needs error handling.

CreateFile
>If the function succeeds, the return value is an open handle to the specified file, device, named pipe, or mail slot.
>If the function fails, the return value is INVALID_HANDLE_VALUE. To get extended error information, call [GetLastError](https://learn.microsoft.com/en-us/windows/desktop/api/errhandlingapi/nf-errhandlingapi-getlasterror).

Probably doesn't hurt to call close INVALID_HANDLE_VALUE, but probably better not to.